agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -1710,8 +1721,8 @@ async def create_new_evaluation_scenario(
         note=note,
         evaluators_configs=evaluators_configs,
         results=results,
-        created_at=datetime.utcnow(),
-        updated_at=datetime.utcnow(),
+        created_at=datetime.now(),
```

Comment:
I thought previously that this was automatically populated (date), no?

Reply:
Yes, yes. I'll take it out from the function. Thank you for noticing that.